File: templates/matlab/microgrid_market_contribution_bridge.m
function fig = microgrid_market_contribution_bridge()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('waterfall', 3808, 'microgrid and market analysis: contribution waterfall', 'microgrid and market analysis', 'contribution waterfall');
end

File: templates/python/microgrid_market_contribution_bridge.py
"""microgrid_market_contribution_bridge: 微电网与市场贡献瀑布桥（waterfall 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='microgrid and market analysis: contribution waterfall'):
    return make_template_figure('waterfall', seed=3808, title=title, domain='microgrid and market analysis', topic='contribution waterfall')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
